### ISSUE TITLE
Contacting Le Wang

### ISSUE THREAD

**Issue by spwhitton** [lewang/ws-butler#52]
Hello,

I have been trying to get in touch with the author via the e-mail address on the git commits, but have not received any response.  Le, if you see this, can you write to spwhitton@spwhitton.name?  Or if anyone else knows how to contact Le, please ask him to do that.

Thanks!

**Comment by lewang** [comment]
can I help with something?

**Comment by spwhitton** [comment]
@lewang Hey!  I sent you an e-mail.  Here is what I wrote:

> Hello Le,
> 
> I love ws-butler, as do many others.
> 
> ws-butler has a number of open issues and could generally use some
> maintainance work.
> Would you consider letting me take over as maintainer, please?
> 
> I've been maintaining ws-butler in the official Debian repositories
> since 2016 and I have commit access to upstream Emacs, GNU ELPA and
> NonGNU ELPA.
> 
> I'd leave you as Author, and add a Maintainer line for myself.
> You could either grant me access to your repository on github, or
> transfer the repository to me, which automatically establishes a
> redirect.  In the latter case I could give you push access so you could
> still make changes.
> 
> Thank you for considering this.
> 
-- 
Sean Whitton

**Comment by spwhitton** [comment]
I've taken over maintenance of the version of ws-butler distributed by NonGNU ELPA.  I've released version 1.0 there.

**Comment by spwhitton** [comment]
> Melpa still points here and this repo is not marked as unmaintained. Was that decision unilateral?

Le Wang has never responded.  I think "unilateral" is not the right word, but it's true that a decision was taken in the absence of any communication from the maintainer of this particular git repository.  The NonGNU ELPA maintainers, which includes me, decided to start maintaining a new branch of ws-butler.  MELPA can (and hopefully will) make the same decision, but it's up to them.

**Comment by lewang** [comment]
I mean I could add you as a contributor?  Transfer the repo to you on github?

**Comment by spwhitton** [comment]
Hey!  Yes, I think transferring the repository to me on GitHub would
minimise user confusion because then lewang/ws-butler would redirect to
spwhitton/ws-butler.
-- 
Sean Whitton

**Comment by oscarfv** [comment]
Why not moving the project to [emacs-orphanage](https://github.com/emacsorphanage) and closing this repo with a link to it?

AFAIK, emacs-orphanage does not depend on a single individual.

**Comment by oscarfv** [comment]
Fair enough, but active maintainers come and go. Putting the project under an umbrella that keeps it active regardless of $LIFE is a good idea in general. Moving it to GNU/ELPA/Emacs core would be good too.

Anyway, thank you for taking over the maintenance of this nice little package that keeps my OCD at bay :-)

**Comment by lewang** [comment]
I was thinking about this.  If I transfer to @spwhitton , and something happens (say he burns out), we'd be in a similar situation.

But reading the docs on emacsorphanage, it seems that's intended for packages without a maintainer.

**Comment by spwhitton** [comment]
As it's in NonGNU ELPA now, other Emacs committers would probably take
it over.

-- 
Sean Whitton

**Comment by lewang** [comment]
I have a channel to talk to @spwhitton .  I'm going to leave the transfer issue for now.
